chaco/array_plot_data.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, *data, **kw):
         conventions, but was it chosen for convenience, since the raison d'etre
         of this class is convenience.
         """
-        super(AbstractPlotData, self).__init__()
+        super(ArrayPlotData, self).__init__()
```

Comment:
Since we are only supporting Python 3+ now, we can use `super()` and avoid these problems entirely.

Reply:
that is the final plan - i wanted to make smaller changes incrementally to ensure that the behavior is still the same.